src/tests/utils.ts

```diff
@@ -0,0 +1,58 @@
+import * as monaco from 'monaco-editor'
+import { ISingleEditOperationIdentifier, ValidAnnotatedEditOperation } from 'vscode/vscode/vs/editor/common/model'
+import { EDITOR_DEFAULT_CODE } from './constants'
+
+export function createTestModel (content: string, language: string = 'typescript'): monaco.editor.ITextModel {
+  return monaco.editor.createModel(content, language)
+}
+
+export function createDefaultTestModel (): monaco.editor.ITextModel {
+  return createTestModel(EDITOR_DEFAULT_CODE)
+}
+
+export function createTestRange (
+  model: monaco.editor.ITextModel,
+  startLine: number,
+  endLine: number,
+  startColumn: number = 1,
+  endColumn?: number
+): monaco.Range {
+  if (endColumn == null) {
+    endColumn = model.getLineMaxColumn(endLine)
```

Comment:
nit: you can use it as default value line 18 (:

Reply:
[test: simplify createTestRange function](https://github.com/CodinGame/monaco-editor-wrapper/pull/125/commits/8ec243327bd2e8a240bd3ab1e01f1207a3c3e81d)